dist/main.bundle.js

```diff
@@ -314,7 +314,7 @@ eval("var d3 = __webpack_require__(/*! d3 */ \"./node_modules/d3/index.js\");\nv
v) {\n                return UTIL.getExpressionConfig(v, ['color']);\n            });\n            return chart;\n        }\n\n        var index = _measure.indexOf(measure);\n\n        if (index === -1) {\n            throw new Error('Invalid measure provided');\n        }\n\n        if (value == void 0) {\n            return _textColorExpressionForMeasure[index];\n        } else {\n            _textColorExpressionForMeasure[index] = UTIL.getExpressionConfig(value, ['color']);\n        }\n    }\n\n    chart.iconNameForMeasure = function (value, measure) {\n        return _baseAccessor.call(_iconNameForMeasure, value, measure);\n    }\n\n    chart.iconPositionForMeasure = function (value, measure) {\n        return _baseAccessor.call(_iconPositionForMeasure, value, measure);\n    }\n\n    chart.iconExpressionForMeasure = function (value, measure) {\n        if (!arguments.length) {\n            /**\n             * Getter method call with no arguments\n             * E.g. <chart>.kpiIconExpression() ==> [<item1>, <item2>]\n             */\n            return _iconExpressionForMeasure;\n        }\n\n        if (value instanceof Array && measure == void 0) {\n            /**\n             * Setter method call with only value argument\n             * E.g. <chart>.kpiIconExpression([<item1>, <item2>]) ==> <chart_function>\n             */\n            _iconExpressionForMeasure = value.map(function (v) {\n                return UTIL.getExpressionConfig(v, ['icon', 'color']);\n            });\n            return chart;\n        }\n\n        var index = _measure.indexOf(measure);\n\n        if (index === -1) {\n            throw new Error('Invalid measure provided');\n        }\n\n        if (value == void 0) {\n            /**\n             * Getter method call with only measure argument\n             * E.g. <chart>.kpiIconExpression(<measure>) ==> <item>\n             */\n            return _iconExpressionForMeasure[index];\n        } else {\n            /**\n             * Setter method call with both value and measure arguments\n             * E.g. <chart>.kpiIconExpression(<item>, <measure>) ==> <chart_function>\n             */\n            _iconExpressionForMeasure[index] = UTIL.getExpressionConfig(value, ['icon', 'color']);\n        }\n\n        return chart;\n    }\n\n    chart.fontWeightForMeasure = function (value, measure) {\n        return _baseAccessor.call(_fontWeightForMeasure, value, measure);\n    }\n\n    chart.iconFontWeight = function (value, measure) {\n        return _baseAccessor.call(_iconFontWeight, value, measure);\n    }\n\n    chart.iconColor = function (value, measure) {\n        return _baseAccessor.call(_iconColor, value, measure);\n    }\n\n    chart.print = function (value) {\n        if (!arguments.length) {\n            return _print;\n        }\n        _print = value;\n        return chart;\n    }\n\n    return chart;\n}\n\nmodule.exports = table;\n\n//# sourceURL=webpack://flairVisualizations/./js/charts/table.js?");
```

Comment:
@khushbu-maurya : Why do we have an eval() function here? Can you please check and let me know? Thanks

Reply:
@admin-vizcentric its auto-generated code by webpack